cmake/package_dragndrop_dmg.cmake

```diff
@@ -92,29 +88,6 @@ add_custom_target(dist
   COMMENT "Building ${CPACK_GENERATOR} package"
 )
 
-########################################################### Create dmg with background image
-if (DEFINED CMAKE_VERSION AND NOT "${CMAKE_VERSION}" VERSION_LESS "3.5")
-  set(OPENMS_LOGO ${PROJECT_SOURCE_DIR}/cmake/MacOSX/openms_logo_large_transparent.png) ## The logo to be used for the OpenMS folder on the DMG
-  set(OPENMS_DMG_FOLDER_NAME "${CPACK_PACKAGE_NAME}-${OPENMS_PACKAGE_VERSION_FULLSTRING}") ## The name of the OpenMS folder on the DMG
-  configure_file(${PROJECT_SOURCE_DIR}/cmake/MacOSX/setup_applescript.scpt.in ${PROJECT_BINARY_DIR}/macOS_bundle_setup/setup_applescript.scpt)
-  set(CPACK_DMG_DS_STORE_SETUP_SCRIPT ${PROJECT_BINARY_DIR}/macOS_bundle_setup/setup_applescript.scpt)
-  #Next line could overcome a script but since we do not have a fixed name of the OpenMS-$VERSION folder, it probably won't work
-  #set(CPACK_DMG_DS_STORE ${PROJECT_SOURCE_DIR}/cmake/MacOSX/DS_store_new)
-  set(CPACK_DMG_BACKGROUND_IMAGE ${PROJECT_SOURCE_DIR}/cmake/MacOSX/background.png)
-  set(CPACK_DMG_FORMAT UDBZ) ## Try bzip2 to get slighlty smaller images
```

Comment:
This would be really nice to keep.

Reply:
My takeaway from our discussion yesterday was that DMG signing was not feasible at present, and Apple was likely to only make it harder in the future. I can add a comment leaving breadcrumbs to this issue if you think that would help.

Reply:
But this specific part is for designing the DMG style. Otherwise it is really ugly.

Reply:
We have to use DMG until someone implements PKG

Reply:
Okay. Does this look better now?